src/control/cmd/daos/snapshot.go

```diff
@@ -18,8 +18,9 @@ import "C"
 type containerSnapshotCreateCmd struct {
 	existingContainerCmd
 
-	Epoch uint64 `long:"epc" short:"e" description:"epoch to use for snapshot"`
-	Name  string `long:"snap" short:"s" description:"snapshot name"`
+	// FIXME: epoch is not an input for snapshot create
```

Comment:
If this shouldn't be accepted by the create command, probably best to just remove it now.

Reply:
+1

Reply:
Removed